distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -120,10 +121,12 @@ type Column
                 case needs_polyglot_conversion of
                     True -> Java_Column.fromItems name (enso_to_java_maybe items) expected_storage_type java_problem_aggregator
                     False -> Java_Column.fromItemsNoDateConversion name items expected_storage_type java_problem_aggregator
-            result = Column.from_java_column java_column . throw_on_warning Conversion_Failure
```

Comment:
Calling `Any.throw_on_warning` extract the first value of an _intersection type_ of a multi value that has such a method - e.g. `Column` and invokes the method on such a _simple value_ while loosing the additional types.

- #11827
- #11935

The workaround is to avoid calling `Any` instance methods. Done in 9bb73aec68313e83169f29dd785604a5b933d00c